src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface ReportService {
+    public String writeToString();
```

Comment:
```suggestion
    public String createReport();
```
let's make this method name more clear